docs/sql-ref-syntax-qry-select.md

```diff
@@ -85,6 +85,7 @@ SELECT [ hints , ... ] [ ALL | DISTINCT ] { named_expression [ , ... ] }
      * [Table-value function](sql-ref-syntax-qry-select-tvf.html)
      * [Inline table](sql-ref-syntax-qry-select-inline-table.html)
      * Subquery
+     * [File](sql-data-sources-load-save-functions.html#run-sql-on-files-directly)
```

Comment:
Could you create a new page like the above clauses?

Reply:
@liaoaoyuan97 Could you address the comment above? I think it would be nicer to place the section "Run SQL on files directly" in the SQL references.

Reply:
Created: https://github.com/apache/spark/pull/30165